File: src/main/java/com/kingmang/ixion/api/Debugger.java
package com.kingmang.ixion.api;

public class Debugger {
    public static boolean DEBUG = false;
    private static final String BLUE_START = "\u001B[34m";
    private static final String RESET = "\u001B[0m";

    public static void debug(String message) {
        if (DEBUG) {
            System.out.println(BLUE_START.concat(message).concat(RESET));
        }
    }
}

File: src/main/java/com/kingmang/ixion/api/Debugger.kt
package com.kingmang.ixion.api

object Debugger {
    var DEBUG: Boolean = false
    private const val BLUE_START = "\u001B[34m"
    private const val RESET = "\u001B[0m"

    @JvmStatic
    fun debug(message: String) {
        if (DEBUG) {
            println(BLUE_START + message + RESET)
        }
    }
}
